Cargo.toml

```
@@ -68,8 +68,10 @@ exclude = [
 actix-web = "4"
 anyhow = "1.0.66"
 lazy_static = "1.4.0"
+reqwest = "0.11"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.85"
+tokio = { version = "1.28", features = ["rt", "rt-multi-thread", "sync"] }
```

```suggestion
tokio = "1.28"
```

This is no longer necessary given how we implemented the tokio executor on a separate thread.

---

Thank you! I already remove it.